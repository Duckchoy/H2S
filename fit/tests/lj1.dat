0 0.000175813389875      -0.240403998522525      0.004059737214175      -1.240320306123500      -19469.0616179229      0.238686602138741
0 0.000092182491025      -0.183442153938825      0.001734021272225      -0.879631675452150      -19472.9120892096      0.182401643159099
0 0.000049860284075      -0.142181075095025      0.000789655381475      -0.640647235786800      -19473.4094089342      0.182166503514693
0 0.000015916676050      -0.089042888161775      0.000192134472075      -0.362916405310925      -19473.6622826838      0.154010797385904
0 0.000009362139950      -0.071771090191525      0.000101348203250      -0.280735363729050      -19473.2305595504      0.177090111454995
0 0.000001752986425      -0.036506867970250      0.000014190657850      -0.127642768154075      -19471.3645007981      0.246164621207460
