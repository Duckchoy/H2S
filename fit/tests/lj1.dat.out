prob -5.272550519

 new best -19467.1012 669.2441706 329.4971671 -151.8662094 -62.69403786 -5.272550519
prob -3.734025516

 new best -19468.05759 19.5032205 264.8556599 281.3777479 -49.50786058 -3.734025516
prob -4.547453376
prob -5.147063627
prob -5.043077132
prob -3.776273012
prob -3.763246532
prob -3.709859945

 new best -19468.09301 -475.5749976 262.1199751 326.4855366 -48.93521723 -3.709859945
prob -4.400913141
prob -3.875328154
prob -3.656382692

 new best -19468.59917 -127.180442 216.4580482 788.2924831 -38.9792634 -3.656382692
prob -3.722740792
prob -3.796511349
prob -4.143760345
prob -16.30581904
prob -10.77590528
prob -5.15367054
prob -3.646474181

 new best -19468.40868 -42.30560721 231.237937 654.518536 -42.11157009 -3.646474181
prob -4.715437947
prob -4.540709953
prob -4.126092718
prob -5.503044649
prob -6.976219086
prob -22.09780037
prob -4.305956311
prob -11.57027852
prob -6.66316962
prob -4.324223024
prob -4.280061343
prob -3.659471551
prob -3.730013541
prob -6.385280918
prob -3.836749859
prob -4.915209613
prob -5.001885777
prob -4.347856348
prob -4.709508025
prob -5.366327084
prob -5.089371504
prob -4.733340711
prob -5.404142679
prob -4.130818248
prob -4.386542725
prob -4.108544703
prob -4.721380825
prob -3.713836433
prob -3.750136666
prob -20.44140349
prob -3.827657358
prob -4.393080179
prob -5.00677868
prob -6.878218509
prob -4.709112917
prob -12.96754135
prob -3.964160695
prob -4.099947179
prob -4.746736528
prob -3.837892726
prob -3.709498024
prob -4.704078939
prob -4.035688529
prob -5.8725829
prob -4.327091905
prob -3.970781287
prob -4.708968035
prob -8.949164211
prob -4.12345857
prob -4.467187557
prob -5.67537561
prob -3.847188732
prob -9.419493614
prob -5.862165398
prob -4.82933793
prob -5.748706843
prob -4.524340154
prob -4.262926802
prob -3.851918414
prob -4.940745777
prob -5.160476589
prob -4.330853079
prob -4.700305202
prob -8.730664813
prob -6.48383771
prob -6.06356111
prob -6.274877239
prob -3.742195623
prob -3.690203723
prob -4.837439489
prob -5.197676441
prob -4.418431033
prob -4.235233635
prob -6.212989179
prob -4.981615589
prob -4.75587537
prob -4.664300684
prob -5.344940677
prob -3.753026274
prob -5.023698617
prob -4.937264431
prob -5.976463563
prob -3.632128027

 new best -19468.31092 -865.9000516 243.5061215 526.9268426 -44.92315252 -3.632128027
prob -3.829202451
prob -4.060988069
prob -3.759483688
prob -3.669709984
prob -3.829704765
prob -4.557806673
prob -4.609184883
prob -3.726832024
prob -4.785107899
prob -4.003972325
prob -6.293952697
prob -4.692553585
prob -4.984026439
prob -6.078329954
prob -4.867738284
prob -4.723807629
prob -5.648592539
prob -3.846854514
prob -3.940108357
prob -5.3745818
prob -5.273390989
prob -4.856827516
prob -5.883419025
prob -3.838651587
prob -4.21872918
prob -4.735760026
prob -4.93691844
prob -3.680099228
prob -13.89142586
prob -4.297430311
prob -17.21313437
prob -4.648192034
prob -5.141192254
prob -16.87672009
prob -3.91949473
prob -3.644858952
prob -18.56770285
prob -3.821775411
prob -14.89591871
prob -3.671513254
prob -13.60081213
prob -8.890672573
prob -4.342724676
prob -5.088764595
prob -4.699948413
prob -4.0259219
prob -10.55344579
prob -4.618662314
prob -3.814657186
prob -6.797219236
prob -4.645446564
prob -3.759693648
prob -4.430639869
prob -6.36278193
prob -4.066977055
prob -7.773352829
prob -3.834560489
prob -5.029236079
prob -5.59604472
prob -3.676572504
prob -4.147151054
prob -4.048684012
prob -5.375910319
prob -4.549163055
prob -4.809930579
prob -3.826194631
prob -3.870341997
prob -3.985548863
prob -4.761456099
prob -4.070593905
prob -3.76031069
prob -4.787018185
prob -6.257361851
prob -6.14876291
prob -14.19739869
prob -8.683464652
prob -3.656861856
prob -4.190472101
prob -4.643832661
prob -4.626868039
prob -6.985363544
prob -4.999685663
prob -4.119893573
prob -3.67695097
prob -3.678598532
prob -4.536843494
prob -4.833195337
prob -3.920502694
prob -6.459780885
prob -4.697836581
prob -4.09252992
prob -5.59900175
prob -6.001007176
prob -3.752124078
prob -3.851167323
prob -3.929163509
prob -4.338081859
prob -4.035370468
prob -4.771238964
prob -5.072632814
prob -6.116406568
prob -3.751391678
prob -4.422164284
prob -3.919220412
prob -12.63053989
prob -6.634600437
prob -17.95931073
prob -5.079740349
prob -4.229597694
prob -4.344522922
prob -6.014410329
prob -4.328839181
prob -4.558435352
prob -5.511212882
prob -3.64682073
prob -3.702947827
prob -4.956739333
prob -4.929695356
prob -3.806808564
prob -4.227241964
prob -3.65718965
prob -3.643073869
prob -15.84393907
prob -6.365738034
prob -3.966280487
prob -4.185506105
prob -3.799247611
prob -4.168508388
prob -4.000692655
prob -4.703632406
prob -6.574455258
prob -4.334300992
prob -3.725504012
prob -3.789652494
prob -3.678787603
prob -4.387971068
prob -3.688086115
prob -4.79382759
prob -3.637901927
prob -4.427955533
prob -4.514147346
prob -5.758250308
prob -4.056562615
prob -3.679475062
prob -5.795101012
prob -7.169169751
prob -11.81442007
prob -9.916084859
prob -5.640873958
prob -6.405157229
prob -3.779117057
prob -5.081767618
prob -5.68353126
prob -5.690685661
prob -3.626311075

 new best -19468.49216 491.6570122 227.7789623 626.549272 -41.4974176 -3.626311075
prob -9.787281782
prob -6.725875924
prob -4.533526403
prob -5.374382784
prob -3.710728105
prob -6.587101118
prob -3.925244032
prob -4.592387035
prob -4.618069408
prob -18.14024887
prob -3.661080075
prob -5.535328079
prob -3.818170229
prob -5.52398393
prob -3.807755751
prob -3.677065096
prob -4.742942313
prob -3.9857152
prob -6.040813568
prob -3.770466677
prob -4.893632476
prob -5.759868813
prob -4.250132296
prob -13.51291141
prob -4.14901062
prob -7.16807943
prob -3.657458292
prob -4.154429038
prob -4.694394846
prob -7.131385213
prob -4.262085863
prob -4.579559146
prob -3.822552261
prob -4.444772511
prob -3.76275265
prob -5.204360389
prob -4.398050435
prob -4.606860152
prob -6.885840033
prob -4.807771523
prob -3.682196771
prob -3.698516856
prob -7.802597212
prob -4.30615021
prob -8.824997619
prob -4.49874805
prob -10.49543722
prob -3.652283916
prob -4.318303447
prob -3.802183071
prob -4.473189312
prob -5.552100835
prob -4.741939405
prob -6.6362808
prob -3.885360319
prob -4.249075241
prob -3.649550809
prob -4.175440892
prob -4.025760024
prob -3.646900256
prob -4.222681988
prob -4.627109029
prob -6.905841837
prob -6.190331176
prob -5.119308956
prob -5.534998231
prob -3.999343995
prob -8.557940602
prob -6.285196921
prob -5.32177339
prob -3.756412805
prob -3.957179754
prob -3.837959821
prob -3.764036793
prob -4.315495397
prob -5.25797523
prob -4.586352843
prob -8.946414328
prob -14.48680387
prob -3.691947582
prob -4.501506827
prob -4.064853546
prob -7.669871797
prob -3.812701447
prob -5.270875135
prob -8.678245668
prob -6.83865767
prob -5.029335678
prob -6.582429794
prob -4.932391402
prob -4.45368335
prob -5.732530864
prob -3.635710922
prob -4.786201304
prob -4.120494941
prob -5.050752084
prob -6.729378071
prob -4.320840334
prob -3.850824809
prob -3.720930457
prob -4.926368388
prob -3.795977743
prob -5.747214482
prob -6.268576508
prob -5.575398837
prob -6.878631397
prob -3.809596501
prob -4.157820868
prob -3.903549365
prob -3.619527235

 new best -19468.42875 -621.6761431 233.4756275 610.5031766 -42.76334478 -3.619527235
prob -7.983996996
prob -4.701869075
prob -3.691587863
prob -5.371605527
prob -7.695806565
prob -4.667220136
prob -5.385898206
prob -4.383383921
prob -4.036717484
prob -4.203615657
prob -3.793302963
prob -4.864906591
prob -4.236515607
prob -19.67619922
prob -4.491069672
prob -3.72665912
prob -5.128317014
prob -3.936415193
prob -4.167232314
prob -3.632415662
prob -3.705185055
prob -4.758868192
prob -6.973339872
prob -12.55372725
prob -3.830413607
prob -10.36210899
prob -6.853314966
prob -3.76369916
prob -16.48353927
prob -5.220677093
prob -4.883519545
prob -4.22397915
prob -13.94293008
prob -5.765693363
prob -4.663405015
prob -4.308902252
prob -3.858799731
prob -3.709825301
prob -7.016468333
prob -5.180211052
prob -4.801790269
prob -4.091838986
prob -4.462919924
prob -3.663644792
prob -4.449322247
prob -3.719411386
prob -5.345538741
prob -4.111336884
prob -3.668563137
prob -4.852742787
prob -12.18881553
prob -4.525989858
prob -3.711579006
prob -3.941351584
prob -5.317818279
prob -12.95150182
prob -4.181501173
prob -4.890001844
prob -4.411652176
prob -3.700772377
prob -3.648119828
prob -4.874821038
prob -3.659361473
prob -3.826550769
prob -4.64741038
prob -4.938245409
prob -9.60192385
prob -3.939911641
prob -4.599141024
prob -24.42683954
prob -3.872529189
prob -3.781755322
prob -5.22086031
prob -4.304024086
prob -3.757384825
prob -3.757283931
prob -4.031173727
prob -3.805064866
prob -4.514806251
prob -14.27096165
prob -5.296908222
prob -4.456231729
prob -3.875531317
prob -17.08949247
prob -3.658624925
prob -4.485884841
prob -3.71082048
prob -7.192773951
prob -3.988965401
prob -4.553353582
prob -4.528809842
prob -4.227975713
prob -11.51715929
prob -3.691160389
prob -3.649980482
prob -4.099921502
prob -3.925913023
prob -13.63230887
prob -4.103068042
prob -8.026610319
prob -10.91442174
prob -13.93230319
prob -3.627260432
prob -3.829165379
prob -5.245327851
prob -4.349043852
prob -3.772571154
prob -5.005366614
prob -15.86313555
prob -6.678989182
prob -6.241898897
prob -3.796090474
prob -5.687368319
prob -4.17973209
prob -6.068723857
prob -17.45002019
prob -6.668699167
prob -3.940575879
prob -4.142786878
prob -5.694340289
prob -5.348725459
prob -5.750989236
prob -5.564682015
prob -9.187476379
prob -5.542121283
prob -5.622598264
prob -5.389826445
prob -4.761054908
prob -6.494310009
prob -4.439631684
prob -3.626522334
prob -3.863245092
prob -3.684847328
prob -3.91528928
prob -3.637760459
prob -7.796755649
prob -5.790291124
prob -3.955173281
prob -4.321326661
prob -6.396514845
prob -4.234818968
prob -3.901032431
prob -4.790209742
prob -4.530947411
prob -5.078596164
prob -4.16205321
prob -4.273232944
prob -6.211451304
prob -4.109353648
prob -5.646113093
prob -3.728295595
prob -3.771533822
prob -3.937932364
prob -11.48219244
prob -3.952494061
prob -3.83650336
prob -4.219312334
prob -3.627335577
prob -4.551999512
prob -4.366825713
prob -3.788940995
prob -5.67822975
prob -6.41143033
prob -9.691884038
prob -3.639996228
prob -3.628885832
prob -4.484666838
prob -3.783516772
prob -14.89166979
prob -3.80630861
prob -20.37625593
prob -6.352068605
prob -3.75262159
prob -3.629871516
prob -3.647116448
prob -6.256333217
prob -4.682223205
prob -4.815442125
prob -3.628830093
prob -4.073082584
prob -6.865983068
prob -4.627282308
prob -9.180975853
prob -5.367889571
prob -3.880935734
prob -4.577135151
prob -3.806818263
prob -3.844650989
prob -3.791529068
prob -4.33257306
prob -3.9065573
prob -3.805936638
prob -4.155206603
prob -5.297799664
prob -4.3207735
prob -6.683429964
prob -4.083496077
prob -4.988875645
prob -3.674758443
prob -3.674736601
prob -7.486234477
prob -3.639843329
prob -3.629220924
prob -14.57704251
prob -10.47323337
prob -7.286167552
prob -3.889719437
prob -20.7082377
prob -6.100459632
prob -4.04683996
prob -5.288446107
prob -5.394384363
prob -3.871540125
prob -4.261478399
prob -5.5081393
prob -3.992734168
prob -3.870314193
prob -4.450393669
prob -4.136634156
prob -4.4051162
prob -4.565129145
prob -3.881403201
prob -3.996815912
prob -5.739179503
prob -4.022477667
prob -10.09082163
prob -7.702721092
prob -3.641497643
prob -3.796967153
prob -6.012239438
prob -4.330844006
prob -3.82337494
prob -4.285789471
prob -3.643635979
prob -4.267306414
prob -19.90053833
prob -4.209827879
prob -4.748216754
prob -5.723841656
prob -4.14009756
prob -3.994845178
prob -9.937180899
prob -6.125937404
prob -4.029641706
prob -4.038682965
prob -3.81834027
prob -3.671449587
prob -7.077309271
prob -3.670166074
prob -5.292721153
prob -5.452606015
prob -5.687760616
prob -3.994151155
prob -5.31579899
prob -6.225648024
prob -5.580741891
prob -4.015851955
prob -15.90867202
prob -3.901473796
prob -7.135055279
prob -3.653843632
prob -4.832587489
prob -4.396113397
prob -4.520580703
prob -17.36342256
prob -3.647521453
prob -4.266064893
prob -10.24342209
prob -4.037147801
prob -12.15738088
prob -4.200652718
prob -5.139354105
prob -8.70488881
prob -9.590866283
prob -4.114537522
prob -4.059913388
prob -3.993090438
prob -4.956217247
prob -5.037928284
prob -7.750184486
prob -4.468805462
prob -5.804476348
prob -5.960669667
prob -3.722633843
prob -3.9320431
prob -3.72142184
prob -5.480508564
prob -4.140485792
prob -5.506113503
prob -3.742357291
prob -4.429236669
prob -6.253147788
prob -4.709991868
prob -4.541489087
prob -4.905494979
prob -3.99270809
prob -5.611267296
prob -4.253841674
prob -18.98364248
prob -3.632606044
prob -3.805197076
prob -12.88214652
prob -5.183777174
prob -3.650010447
prob -5.377008118
prob -7.724559108
prob -8.131605907
prob -4.719633686
prob -9.638146087
prob -4.002854188
prob -11.80761607
prob -4.389621877
prob -11.09857342
prob -3.993845472
prob -3.636168047
prob -4.586603734
prob -7.342086773
prob -13.45701653
prob -5.786832477
prob -5.801786854
prob -3.717135714
prob -6.596157352
prob -4.627925309
prob -4.866752232
prob -3.832346935
prob -4.511624154
prob -11.62921775
prob -5.013143024
prob -4.436811913
prob -4.59648839
prob -4.42606352
prob -5.821126538
prob -4.609250933
prob -4.783104994
prob -4.296239036
prob -4.185653848
prob -4.555324332
prob -4.344691762
prob -3.776421983
prob -6.581800194
prob -11.54162548
prob -5.976172845
prob -4.484655136
prob -5.098317762
prob -3.896026916
prob -3.705929565
prob -6.545492638
prob -15.24196263
prob -12.1413896
prob -4.521512105
prob -12.60015729
prob -3.732444421
prob -4.999988608
prob -5.270028007
prob -3.860004897
prob -4.685129074
prob -15.17631316
prob -4.154500489
prob -4.802367526
prob -19.3426231
prob -5.8018165
prob -4.276166392
prob -3.79357777
prob -4.407011019
prob -4.02431908
prob -3.758701578
prob -5.502922209
prob -8.876916929
prob -3.734245729
prob -3.840553202
prob -4.0275836
prob -4.232616234
prob -4.735309625
prob -9.917040313
prob -6.577224441
prob -3.859522781
prob -7.195076757
prob -4.949239704
prob -4.605843817
prob -4.95638484
prob -4.582319773
prob -4.240732605
prob -3.67753044
prob -4.393805727
prob -4.166227185
prob -3.997492053
prob -3.75940851
prob -4.494634887
prob -4.726194733
prob -4.240532168
prob -4.911351548
prob -4.592731689
prob -5.153704656
prob -5.310873825
prob -5.546067085
prob -3.701612426
prob -5.57134569
prob -5.475704195
prob -3.991028668
prob -3.793980531
prob -5.476286799
prob -7.10934604
prob -3.638900685
prob -3.86800832
prob -5.60226124
prob -3.753540695
prob -16.56528935
prob -5.036557459
prob -3.859132828
prob -6.26619975
prob -3.685001364
prob -6.529548008
prob -3.870255223
prob -3.782080937
prob -8.100716993
prob -5.606715223
prob -3.636903439
prob -3.736884931
prob -3.941609707
prob -4.988339066
prob -9.995756158
prob -3.725036779
prob -3.625314653
prob -3.75379334
prob -6.853317297
prob -4.381956816
prob -4.06451441
prob -4.240991467
prob -11.4914241
prob -4.885181638
prob -4.711782723
prob -4.10494954
prob -3.679106264
prob -13.78323913
prob -5.27422314
prob -6.669176804
prob -7.562753923
prob -4.529343713
prob -5.21946204
prob -6.808554356
prob -4.729010684
prob -4.683179115
prob -4.003600536
prob -5.108603206
prob -4.073134505
prob -4.951905566
prob -3.772241927
prob -5.050020611
prob -5.814078749
prob -4.805281376
prob -4.349205885
prob -9.055957289
prob -8.024024394
prob -4.322458698
prob -3.707716013
prob -4.284962666
prob -5.398431214
prob -10.08897098
prob -3.91837049
prob -5.259496098
prob -6.353111713
prob -3.633928034
prob -3.773860926
prob -9.800067272
prob -3.709966238
prob -3.663893243
prob -3.789140187
prob -3.95245517
prob -6.846477889
prob -4.160469043
prob -3.81821352
prob -3.739602673
prob -3.980068551
prob -4.155273763
prob -4.229967194
prob -3.764903446
prob -15.83299368
prob -4.417964149
prob -4.086843849
prob -4.6521466
prob -15.87241984
prob -21.17672392
prob -8.12502923
prob -5.462292184
prob -4.65839154
prob -3.996412896
prob -9.23778586
prob -6.965238432
prob -4.400611906
prob -4.878101248
prob -3.666567517
prob -4.335996531
prob -5.848023166
prob -4.419193032
prob -8.789855628
prob -14.82128296
prob -3.782102079
prob -5.082303831
prob -6.214383906
prob -3.776962051
prob -5.054587082
prob -3.762449221
prob -3.756330484
prob -4.124640413
prob -4.606822697
prob -15.52081504
prob -4.593109971
prob -3.837717593
prob -3.729466554
prob -3.758029615
prob -5.115669569
prob -3.637929943
prob -3.782923014
prob -4.779155912
prob -4.384027674
prob -8.908232206
prob -3.847984917
prob -3.905845862
prob -4.622083474
prob -8.139047825
prob -5.364161915
prob -4.153200717
prob -4.27572489
prob -3.876492601
prob -4.256352875
prob -4.83559585
prob -4.261726332
prob -4.331000389
prob -3.816435212
prob -3.736038341
prob -5.284253409
prob -4.120362149
prob -3.964495978
prob -4.043074874
prob -3.759909761
prob -12.60481938
prob -4.132524036
prob -11.38646753
prob -4.843268712
prob -3.703014208
prob -5.081349061
prob -3.665658151
prob -5.507482605
prob -3.82490498
prob -6.030216214
prob -4.777609505
prob -4.564601161
prob -4.301422962
prob -4.303636352
prob -6.083105942
prob -12.72232039
prob -3.645280353
prob -4.097872288
prob -5.079036939
prob -6.704121026
prob -4.765066768
prob -4.129575195
prob -3.695246713
prob -5.048262607
prob -3.851132003
prob -4.251909623
prob -8.244662258
prob -4.192450005
prob -4.120271018
prob -3.792299493
prob -7.256535763
prob -5.175906302
prob -3.85009935
prob -4.775043268
prob -3.805347417
prob -3.720140018
prob -6.581199951
prob -5.976892871
prob -4.405994025
prob -3.784295526
prob -4.371748808
prob -4.615372704
prob -3.812577586
prob -17.04561618
prob -4.354628062
prob -5.360445316
prob -6.594906634
prob -6.697007344
prob -3.639115276
prob -18.44648459
prob -4.018159141
prob -4.052577392
prob -3.673596185
prob -9.794775514
prob -4.685577947
prob -7.576046305
prob -4.338106613
prob -5.430966022
prob -3.665194362
prob -4.872918851
prob -9.558651109
prob -14.80133841
prob -4.690611465
prob -4.686988595
prob -11.27299052
prob -4.012217845
prob -8.827625387
prob -5.133888016
prob -4.039560976
prob -11.63729455
prob -3.736392161
prob -4.443558282
prob -5.034193636
prob -6.922754666
prob -4.131162244
prob -6.024496961
prob -6.796661791
prob -11.45618379
prob -4.417288012
prob -9.66046648
prob -6.641514973
prob -6.242240233
prob -6.032150029
prob -3.777896894
prob -3.640338864
prob -5.750887191
prob -3.654821521
prob -4.729518196
prob -5.088038478
prob -3.653045027
prob -4.451171974
prob -3.924455315
prob -3.90762269
prob -6.232735047
prob -11.320723
prob -4.517292025
prob -6.681222746
prob -19.61530462
prob -14.78364552
numerical derivative 0 analytic 0
numerical derivative 0 analytic 0
numerical derivative 0 analytic 0
numerical derivative 0 analytic 0
numerical derivative 10.9063869 analytic 10.9063869
acceptance 0.714434
timesteps   avg   err
0.009088022176  -19468.89492  0.3629408517
1.019172875  -496.528594  49.35832754
0.239489514  210.8433245  18.98844042
1.018955326  615.5269035  161.0699394
0.006083717324  -38.59851039  3.836121109

c[0] = -19468.89492 +/- 0.3629408517
c[1] = -496.528594 +/- 49.35832754
c[2] = 210.8433245 +/- 18.98844042
c[3] = 615.5269035 +/- 161.0699394
c[4] = -38.59851039 +/- 3.836121109
Assessment of the fit:
fitted value -19469.2964 data -19469.06162 +/- 0.2386866021
fitted value -19472.59844 data -19472.91209 +/- 0.1824016432
fitted value -19473.68352 data -19473.40941 +/- 0.1821665035
fitted value -19473.55062 data -19473.66228 +/- 0.1540107974
fitted value -19473.13367 data -19473.23056 +/- 0.1770901115
fitted value -19471.65746 data -19471.3645 +/- 0.2461646212
minimum skew nan
